terraform/data/user_data.txt

```diff
@@ -0,0 +1,10 @@
+Content-Type: multipart/mixed; boundary="==BOUNDARY=="
+MIME-Version: 1.0
+
+--==BOUNDARY==
+Content-Type: text/cloud-boothook; charset="us-ascii"
+
+# Set Docker daemon option dm.basesize so each container gets up to 50GB
+cloud-init-per once docker_options echo 'OPTIONS="${OPTIONS} --storage-opt dm.basesize=50GB"' >> /etc/sysconfig/docker
```

Comment:
If we're provisioning 200GB EBS volumes, then we should probably set this to 200GB.